lyricsheets/models/karaoke.py

```diff
@@ -14,23 +14,75 @@
 KLine = TypeVar("KLine", bound="KLine")
 
 
+def style_check(func):
+    def style_wrapper(*args):
+        if not args[0].style:
+            raise StyleNotBoundException()
+        return func(*args)
+
+    return style_wrapper
+
+
 @dataclass
 class KChar:
     char: str
     i: int
     sylI: int
     line: KLine
     syl: KSyl
-
     fadeOffset: timedelta = timedelta()
     karaStart: timedelta = timedelta()
     karaEnd: timedelta = timedelta()
 
+    _width: float = 0
+    _left: float = 0
```

Comment:
The style field also seems to have vanished somewhere

Reply:
Fixed